src/app.js

```diff
@@ -9,6 +9,7 @@ import allRouter from "./routes/index";
 import documentation from "./documentation/index";
 import "dotenv/config";
 import notFound from "./controllers/notFound";
+import "./events/eventHandler/eventHandler";
```

Comment:
What's the use of this import in this file?

Reply:
It help to run handlers when server is running, when you don't import it nothing happen to handlers